.gitignore

```diff
@@ -1 +1,10 @@
 /config.json
+*.egg-info
+*.py[co]
+/.coverage
+/.mypy_cache
+/.pytest_cache
+/.tox
+/dist
+/venv*
+.vscode/
```

Comment:
shouldn't need:
- egg-info
- coverage
- pytest_cache (we moved it and changed the name)
- tox
- dist
- /venv* (that only applies to the root)
- .vscode (I don't use vscode)

Reply:
This list is from the pre-commit `.gitignore`. I will trim it then.